benchmark/dataset_operations_benchmark.cpp

```diff
@@ -13,20 +13,20 @@
 using namespace scipp;
 using namespace scipp::core;
 
-enum BoolsGeneratorType { ALTERNATING, FALSE, TRUE };
-
-template <BoolsGeneratorType type = BoolsGeneratorType::ALTERNATING>
-std::vector<bool> makeBools(const scipp::index size) {
-  std::vector<bool> data(size);
-  for (scipp::index i = 0; i < size; ++i)
-    if constexpr (type == BoolsGeneratorType::ALTERNATING) {
-      data[i] = i % 2;
-    } else if constexpr (type == BoolsGeneratorType::FALSE) {
-      data[i] = false;
-    } else {
-      data[i] = true;
-    }
-  return data;
+std::vector<bool> make_bools(const scipp::index size,
```

Comment:
Can you link the implementation from `core/test/dataset_test_common.h` to avoid duplicating, or is that difficult to do?

Reply:
The code is similar. I had considered that but doesn't seem worthwhile for these changes alone. As a later step we should maybe look at code sharing between test code and benchmark code.

Reply:
> As a later step we should maybe look at code sharing between test code and benchmark code.

Yes that would make sense.